backend/app/routes/utils.py

```diff
@@ -3,7 +3,7 @@
 from fastapi import APIRouter
 
 # There must be a unique field in the sort for the cursor paging to work.
-DEFAULT_SORT = ["score desc", "id asc"]
+BEST_MATCH_SORT = ["score desc", "id asc"]
```

Comment:
I faced a dilemma here about whether to rename "Default Sort" to "Best Match" in the code as well. I opted to perform the rename in the code as well, but I'm not entirely convinced it's the best choice. What's your opinion? Should we stick with the name "Default Sort" in the code, or should we use "Best Match Sort" everywhere?

Reply:
Well, a good call to refactor variables to avoid future confusion - just wanted to verify that.

I'd add a comment in the backend why this is now considered a "best match" sort though